app.go

```diff
@@ -99,40 +111,22 @@ func (a *App) GetApp() (*AppInfo, error) {
 	}
 }
 
-// CreateMagicLink receives a CreateMagicLinkBody struct, creating a magic link with provided values
-// returns MagicLink on success, error on failure
+// CreateMagicLink creates a Magic Link for your app.
+//
+// Deprecated: use `Passage.Auth.CreateMagicLink` instead.
 func (a *App) CreateMagicLink(createMagicLinkBody CreateMagicLinkBody) (*MagicLink, error) {
-	res, err := a.client.CreateMagicLinkWithResponse(context.Background(), a.ID, createMagicLinkBody)
+	magicLink, err := a.Auth.CreateMagicLink(createMagicLinkBody)
 	if err != nil {
-		return nil, Error{Message: "network error: failed to create Passage Magic Link"}
+		var passageError PassageError
+		if errors.As(err, &passageError) {
+			return magicLink, Error{
```

Comment:
this mapping (same for all the deprecated methods) is technically a breaking change since it drops the `StatusText` value. but since that value is basically just the generic reason for a status code (unlikely that anyone would be checking this), i figured it would be okay to drop.